File: src/utils/logs.py
import click

def error(msg):
    click.echo(
        click.style('==> ', fg='red') + msg
    )

def chapter(msg):
    click.echo(
        click.style('==> ', fg='green') + msg
    )

def echo(msg):
    # if not ctx.obj['ECHO']:
    #     return
    click.echo(msg)

File: src/internal/utils/logs/logs.go
package logs

import (
	"fmt"

	"github.com/fatih/color"
)

func Error(msg string) {
	color.Red("==> Error %s", msg)
}

func Chapter(msg string) {
	color.Green("==> %s", msg)
}

func Echo(msg string) {
	fmt.Println(msg)
}
